src/parser/regex/constant.ts

```diff
@@ -1,5 +1,7 @@
-export default {
-  url: /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/,
+const REGEXES = {
+  url: /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w\-_]*)?\??(?:[-+=&;%@.\w_]*)#?(?:[.!/\\\w]*))?)/,
```

Comment:
`isURL` vs gibi yardımcı fonksiyonları kullansak daha iyi olmaz mı?

Reply:
Bunun nesne olarak tutulmasının sebebi, son kullanıcıya ön tanımlı olarak bu regexleri kullandırtabilmek.

Kullanıcıdan `regex` için `string` bir değer gelirse, burada bir şeye eşleşip eşleşmediğine bakılıyor. Bu yüzden sunduğumuz API için bu dizayn daha uygun görünüyor.

https://seodo.gitbook.io/muninn/config/regex